package.json

```diff
@@ -24,10 +24,12 @@
     "@swc/cli": "^0.1.57",
     "@swc/core": "^1.3.20",
     "@swc/jest": "^0.2.23",
+    "@typescript-eslint/utils": "^6.13.1",
```

Comment:
Added this for JSDoc types, just helps with writing rules. In the future, would love to convert this entire codebase to TypeScript and use the `@typescript-eslint/parser` as that's what we use internally.